include/circt/Dialect/SV/SVStatements.td

```diff
@@ -437,6 +441,20 @@ def VerbatimOp : SVOp<"verbatim"> {
   ];
 }
 
+def VerbatimFileOp : SVOp<"verbatim.file"> {
```

Comment:
Please split IR additions out to their own patch so we get high visibility on them.

Reply:
I have forked this as a more orthogonal `sv.file` into #973 as discussed above. Forgot to mark this PR as draft afterwards, as this PR rides on the addition of `sv.file` to emit its auxiliary files.